### PR TITLE
doc: update known_issue.rst with Matter issue

### PR DESCRIPTION
**Pull request by doublemis1**
Update known_issue.rst with Matter KRKNWK-18315 issue

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -323,6 +323,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-1 v2-5-0
+
+KRKNWK-18315: SPAKE2+ Verifier is not regenerated when using non-default passcode.
+  When building factory data with a non-default passcode, the SPAKE2+ Verifier is not generated based on the selected passcode value, but uses the default passcode value (20202021).
+
+  **Workaround:** Enable the `CONFIG_CHIP_FACTORY_DATA_GENERATE_SPAKE2_VERIFIER` Kconfig option in the application prj.conf configuration file to generate SPAKE2+ Verfier based on the provided passcode.
```
Comment (LuDuda) [inline doc/nrf/releases_and_maturity/known_issues.rst:331]:
```suggestion
  **Workaround:** Enable the `CONFIG_CHIP_FACTORY_DATA_GENERATE_SPAKE2_VERIFIER` Kconfig option to generate SPAKE2+ Verfier based on the provided passcode.
```

Comment (LuDuda) [reply doc/nrf/releases_and_maturity/known_issues.rst:331]: I don't think it is require to use `prj.conf` file.

**Review by LuDuda — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -323,6 +323,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-1 v2-5-0
+
+KRKNWK-18315: SPAKE2+ Verifier is not regenerated when using non-default passcode.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:328]:
```suggestion
KRKNWK-18315: SPAKE2+ Verifier is not regenerated when using non-default passcode
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -323,6 +323,13 @@ Matter
...
+  When building factory data with a non-default passcode, the SPAKE2+ Verifier is not generated based on the selected passcode value, but uses the default passcode value (20202021).
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:329]:
```suggestion
  When building factory data with a non-default passcode, the SPAKE2+ Verifier is not generated based on the selected passcode value, but uses the default passcode value (``20202021``).
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -323,6 +323,13 @@ Matter
...
+
+  **Workaround:** Enable the `CONFIG_CHIP_FACTORY_DATA_GENERATE_SPAKE2_VERIFIER` Kconfig option to generate SPAKE2+ Verfier based on the provided passcode.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:331]:
```suggestion
  **Workaround:** Enable the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_GENERATE_SPAKE2_VERIFIER` Kconfig option to generate the SPAKE2+ Verfier based on the provided passcode.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by markaj-nordic — APPROVED** [review]

**Pull request merged**